parm/config/gefs/config.fetch

```diff
@@ -0,0 +1,38 @@
+#! /usr/bin/env bash
+
+########## config.fetch ##########
+
+echo "BEGIN: config.fetch"
+
+# Get task specific resources
+source "${EXPDIR}/config.resources" fetch
+
+export ICSDIR="@ICSDIR@" # User provided ICSDIR; blank if not provided
+export BASE_IC="@BASE_IC@" # Platform home for staged ICs
+
+export FETCH_YAML_TMPL="${PARMgfs}/stage/master_gefs.yaml.j2"
```

Comment:
This should probably be
```suggestion
export FETCH_YAML_TMPL="${PARMgfs}/fetch/ATM_Cold.yaml.j2"
```

Reply:
N/M.  We don't have a fetch YAML for GEFS.  I think this file should be removed instead.

Reply:
deleting that line from  parm/config/gefs/config.fetch